Tasks/Src/Task_Init.c

```diff
@@ -115,6 +116,16 @@ void Task_Init(void *p_arg) {
             TASK_IDLE_STACK_SIZE);
     
     CAN_Queue_Init();
+
+    // If a contactor is on before we turn it on in critical state, it may have failed and welded shut
+    if(Contactor_GetState(HVHIGH_CONTACTOR) || Contactor_GetState(HVLOW_CONTACTOR)) {
```

Comment:
This should not be in the init task. It should be in the CheckContactor task since that is the task that checks the contactor states.

Reply:
shouldnt it be in `task_critical `instead of `checkcontactor `because `checkcontactor `is initialized once the contactors are closed and `task_checkcontactors` is created

Reply:
* `check_contactor_task` is made in task critical

Reply:
Aren't we getting rid of the critical task as well, like we did with the FaultState task?

Reply:
yes